manual/src/html_processing/src/common.ml.in

```diff
@@ -105,6 +107,8 @@ let update_head ?(search = false) soup =
   |> append_child head;
   create_element "script" ~attributes:["src","navigation.js"]
   |> append_child head;
+  create_element "script" ~attributes:["src", "https://plausible.ci.dev/js/script.js"; "defer data-domain", "ocaml.org"]
```

Comment:
I wouldn't mind a comment documenting what is the plausible script that we are linking here.

Reply:
FWIW, I have a bad feeling to embed any random JavaScript from the Internet into the output of the manual -- it is then a burden for those who use an offline version of it. Instead, any ocaml.org specific stuff that is needed should be done in a post-processing step (or via a configure option or environment variable).

Reply:
Oh, ok. Sorry for my lack of understanding about that. If I understand you correctly, the property that http://caml.inria.fr/distrib/ocaml-4.14/ocaml-4.14-refman-html.tar.gz (and future versions) allow offline browsing of the manual is fulfilled?

Reply:
Indeed, those are the archive version that are not post-processed for `ocaml.org` and directly generated from the manual source.

Reply:
Maybe in term of comment :

> script for ocaml.org's instance of plausible analytics.